src/actions/teams.ts

```diff
@@ -42,11 +42,11 @@ async function getProfilesAndStatusesForMembers(userIds: string[], dispatch: Dis
     const requests: Promise<ActionResult|ActionResult[]>[] = [];
 
     if (profilesToLoad.length) {
-        requests.push(getProfilesByIds(profilesToLoad)(dispatch, getState));
+        requests.push(dispatch(getProfilesByIds(profilesToLoad)));
```

Comment:
It is the preferred way to do it, but we haven't gone through and done a mass refactor of it yet. This is also a better way to do it because you can dispatch regular actions and thunk actions with exactly the same syntax this way